src/loadgenerator/logger.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import sys
+from pythonjsonlogger import jsonlogger
+
+# TODO(yoshifumi) this class is duplicated since other Python services are
+# not sharing the modules for logging.
+class CustomJsonFormatter(jsonlogger.JsonFormatter):
+  def add_fields(self, log_record, record, message_dict):
```

Comment:
@daniel-sanche can we use python-logging library for structured logging into stdout instead of having this json implementation?

Reply:
We could, although it wouldn't capture the timestamp in the exact same way.

What's the purpose of the logging here? Are users expected to see them in CLoud Logging, or is this just for internal use? Why does it need to be JSON formatted?

Reply:
> What's the purpose of the logging here? Are users expected to see them in CLoud Logging, or is this just for internal use? Why does it need to be JSON formatted?

I copied this file exactly as it is from our other Python-based services (e.g. email, recommendation). I thought this is what is preferred in this project in terms of logging in python. I don't have a preference in terms of logging and whether they should be visible, but I think it is ok to be visible in Cloud logging. However, I will defer to you all to decide what loggings we want.

Reply:
Ok, I think we should leave it as-is for this PR then. We could maybe open an issue to re-assess the logging method later, but I'm not sure I know the full scope